### ISSUE TITLE
Can not print without a spool selected in 1.4.0

### ISSUE THREAD

**Issue by The-EG** [OllisGit/OctoPrint-SpoolManager#168]
With the new version, 1.4.0, I can no longer start a print with no spool selected. 

Steps to reproduce:
1. Un-select a spool in the spool manager pane:
    ![image](https://user-images.githubusercontent.com/7342077/119499962-44325e00-bd35-11eb-8630-f1b4eb99538b.png)
2. Select a file and click 'Print'

The UI doesn't respond in anyway, but this error does occur in the javascript console:
```
Uncaught TypeError: result is undefined
    confirmSpoolSelectionBeforeStartPrint http://10.0.0.102/ender-3-v2/static/webassets/packed_plugins.js?87fa31a1:4470
    allowedToPrint http://10.0.0.102/ender-3-v2/static/webassets/packed_plugins.js?87fa31a1:4998
    c http://10.0.0.102/ender-3-v2/static/webassets/packed_libs.js?86afb3b9:13
    fireWith http://10.0.0.102/ender-3-v2/static/webassets/packed_libs.js?86afb3b9:13
    l http://10.0.0.102/ender-3-v2/static/webassets/packed_libs.js?86afb3b9:13
    o http://10.0.0.102/ender-3-v2/static/webassets/packed_libs.js?86afb3b9:13
    sentryWrapped http://10.0.0.102/ender-3-v2/static/webassets/packed_core.js?7846f86e:634
```

Prior versions showed a confirmation dialog, and started the print if the appropriate response was chosen.

OctoPrint 1.6.1, FireFox 88.0.1

**Comment by OllisGit** [comment]
Hi @The-EG,

issue should be solved in latest release 1.4.3.

Please test and give feedback.

Thx, in advance
Olli

**Comment by The-EG** [comment]
Confirmed fix in 1.4.3.  Thanks!
